app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsActivity.java

```diff
@@ -188,19 +188,19 @@ void setButtonColors() {
         switch (state)
         {
             case TODAY:
-                stateString = "Today";
+                stateString = getString(R.string.today);
```

Comment:
It would be nice if you could make the whole sentence in `"xDrip+ Statistics for "+stateString+"` translatable :)